aiida/manage/configuration/__init__.py

```diff
@@ -86,12 +86,14 @@ def load_config(create=False):
     from .config import Config
     from .settings import AIIDA_CONFIG_FOLDER, DEFAULT_CONFIG_FILE_NAME
 
+    filepath = os.path.join(AIIDA_CONFIG_FOLDER, DEFAULT_CONFIG_FILE_NAME)
+
     if IN_RT_DOC_MODE:
         # The following is a dummy config.json configuration that it is used for the
         # proper compilation of the documentation on readthedocs.
         from aiida.manage.external.postgres import DEFAULT_DBINFO
         return Config(
-            '/dev/null', {
+            filepath, {
```

Comment:
Ah hm - I wonder:  What if now someone sets the environment variable to run in RTD mode on their local laptop. Will this overwrite the existing config file?

Perhaps it would be safer to simply use some temporary file path?
Could we perhaps reuse some of the code from here:
https://github.com/aiidateam/aiida-core/blob/d750d513f7adf5126d43a230c9fb378de02f0772/aiida/backends/tests/utils/configuration.py#L50

Reply:
That's what I originally was trying as well, but then it is difficult to control the lifetime of it. I will have to create the tempfile in the `load_config` function, but then cannot control the lifetime. This way we will end up with uncleaned temp files. Frankly, I wanted to get rid of these hacks just for RTD for a long time. I remember you also opened an issue recently with some official API to do what we currently do in the `docs/source/conf.py` for I believe plugins to be able to do the same. I'd rather have this go in now and then fix it properly soon

Reply:
Ok, fine not to merge the functionality.
I still think, though, that we should be sure not to overwrite an existing config file (creating a temporary one without deleting it is preferable).

Are we sure your approach does not overwrite an existing config file?